spring-cloud-gcp-data-spanner/src/test/resources/logback.xml

```diff
@@ -0,0 +1,19 @@
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <logger name="io.grpc" level="info"/>
+  <logger name="io.netty" level="info"/>
+  <logger name="org.springframework" level="info"/>
+  <logger name="org.springframework.cloud" level="debug"/>
+
+  <root level="debug">
+    <appender-ref ref="STDOUT" />
+  </root>
+</configuration>
```

Comment:
New line in the end of each file.
You can configure your IDE to do that for you on save.